services-js/commissions-app/src/client/CommissionsListSection.tsx

```diff
@@ -198,6 +198,7 @@ function renderCommission(
             )}
           </div>
         }
+        // @ts-ignore
```

Comment:
I’d like to avoid `@ts-ignore` in our code. This may be cleared up in latest `develop`, but if not you can tweak the type of Props to fix this.